### DISCUSSION TITLE
Feedback for the project - Assignment 1

### DISCUSSION THREAD

**Discussion by rishi1596** [airavata-courses/neo#74] [General]
Hi,
I checked and followed all the steps mentioned to build the project. Everything works fine.
I have observed that after submitting the request it takes a lot of time to render the map on the dashboard, from the user's perspective it seems like something went wrong on the backend side to process it. 
Just a suggestion, you can add a text on the dashboard mentioning - it might take 5 - 10 minutes to load it.
Apart from that everything works fine and looks great to me.
Thanks.
Feedback by Team Garuda.

<img width="1680" alt="Screen Shot 2022-02-09 at 3 50 40 PM" src="https://user-images.githubusercontent.com/25101796/153289521-5f769853-7514-432d-8095-ec421a493b7d.png">

**Comment by niravraje** [comment]
@rishi1596 That's a good point and yes, a message in addition to the loading animation would be helpful in terms of UX (we can add something like "Generating Visualization" maybe). 

We are aware of the delay in showing the visualization and it seems the issue is because of the default resource limit setting on Docker Desktop. On MacOS (M1), the default memory limit is 2GB. Hence it tries to run all our microservices in that space. We ran our application on a Windows machine with 16 GB RAM and the visualization got generated pretty quickly (less than 30 seconds).

But yes, thanks a lot for the feedback.

We'll also update our README.md with an optional step for the user to increase Docker resource limits for a faster generation of visualizations.
